### ISSUE TITLE
[feat] mnist dataset을 활용한 3층 신경망 학습 구현

### ISSUE THREAD

**Issue by Bbuterfly** [Bbuterfly/Paper_implementation#4]
## Task
---
- mnist dataset을 활용한 3층 신경망 학습 기능을 구현합니다.

## TO-DO
---
- [x] softmax function 구현
- [x] 배치처리 구현
- [x] backpropagation 구현
- [x] activation function 구현
- [x] loss function 구현
